addons/website_mass_mailing/static/src/xml/website_mass_mailing.xml

```diff
@@ -1,14 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates id="template" xml:space="preserve">
-<t t-name="website_mass_mailing.edition.wrapper">
-    <div role="dialog" class="modal-dialog modal-md d-block" id="o_newsletter_popup">
-        <div class="modal-content o_popup_modal_content">
-            <div class="o_popup_modal_body text-center">
-                <div class="o_popup_content_dev">
-                    <div id="wrapper"></div>
+
+    <t t-name="website_mass_mailing.popup">
+        <div class="modal fade o_newsletter_modal" t-attf-data-backdrop="#{isEditMode ? 'false' : 'true'}" role="dialog">
+            <div class="modal-dialog modal-dialog-centered" role="document">
+                <div class="modal-content rounded-0">
+                    <span class="btn btn-default text-center rounded-circle o_popup_close o_not_editable fa fa-close" data-dismiss="modal"/>
```

Comment:
Put the close button inside the modal-body or inside a modal-header, that would be more correct.